### PR TITLE
Allow for lower-left-corner map labels to appear for newly added metrics

### PR DESCRIPTION
**Pull request by runforyourlives**
filepath is deprecated as part of the conditions.json config, and new metrics don't have it.  However, this was preventing the map labels for new metrics from showing up.

Remove filepath as part of the equation for lower-left-corner map labels.

Update test for map label generation to reflect the removal of filepath and include more test cases.

### CODE REVIEW

File: src/interface/src/app/stringify-map-config.pipe.ts [hunk 1]
```diff
@@ -18,13 +18,11 @@ export class StringifyMapConfigPipe implements PipeTransform {
...
     let labels: string[] = [];
-    if (
-      !!mapConfig.dataLayerConfig.filepath &&
-      mapConfig.dataLayerConfig.filepath.length > 0
-    ) {
-      let dataLabel = mapConfig.dataLayerConfig.display_name
-        ? mapConfig.dataLayerConfig.display_name
-        : mapConfig.dataLayerConfig.filepath;
+
+    if (!!mapConfig.dataLayerConfig.display_name &&
```
Comment (sachiyuan) [inline src/interface/src/app/stringify-map-config.pipe.ts:22]: Do we want to base this conditional off of if there is a valid display_name or if there is a valid layer? (Aka do we want the display name to appear if the layer isn't properly set?)

File: src/interface/src/app/stringify-map-config.pipe.spec.ts [hunk 2]
```diff
@@ -19,7 +19,89 @@ describe('StringifyMapConfigPipe', () => {
       expect(transformedStr).toEqual('');
     });
 
-    it('should return formatted config string', () => {
+    it('should still return empty string due to lack of a display name and boundary name', () => {
+      let mapConfig: MapConfig = {
+        baseLayerType: BaseLayerType.Road,
+	boundaryLayerConfig: {
+          display_name: 'HUC-12',
```
[the remark this answers is not in the document]
Comment (runforyourlives) [reply src/interface/src/app/stringify-map-config.pipe.spec.ts:26]: Changed to note that it was the boundary's boundary_name needed and the dataLayerConfig's display_name that was needed.

File: src/interface/src/app/stringify-map-config.pipe.ts (hunk 1 above)
Comment (runforyourlives) [reply src/interface/src/app/stringify-map-config.pipe.ts:22]: it's a good question - we don't have a true way to vet that a layer's data is completely valid.  That said, if there is no display name, there's definitely no name to display.  I'm ok considering a layer invalid in this case if it lacks a name but has all other data.

With that, I'll leave a TODO in conditions.json noting that we don't have anything that properly vets whether layers (any of the 3) are complete.

**Review by sachiyuan — APPROVED** [review]
LGTM

**Pull request merged**